README.md

```diff
@@ -30,12 +30,14 @@ configure the Betamocks `enabled`, `cache_dir`, and `services_config` settings:
 - __enabled__: globally turn Betamocks on `true` or off `false`
 - __cache_dir__: the location Betamocks will save its cached response YAML files
 - __services_config__: a YAML file that describes which services and endpoints to mock.
+- __mode__: Either `RECORDING` or `PLAYBACK`.  In `RECORDING`, unmatched requests are sent out and responses recorded as new mock data.  In `PLAYBACK`, unmatched requests fall back to a default response defined in `default.yml`.
 
 ``` ruby
 Betamocks.configure do |config|
   config.enabled = true
   config.cache_dir = File.join(Rails.root, 'config', 'betamocks', 'cache')
   config.services_config = File.join(Rails.root, 'config', 'betamocks', 'betamocks.yml')
+  config.mode = Betamocks::Configuration::RECORDING
```

Comment:
What happens in a deployed environment? is it always hard coded as `Betamocks::Configuration::PLAYBACK`? is there a case where we'd want to toggle this from an ENV var? I guess you could use a string but maybe this could be reworded to accept a boolean then you wouldn't need the constants.